salt/modules/openvswitch.py

```diff
@@ -0,0 +1,302 @@
+# -*- coding: utf-8 -*-
+"""
+Support for Open vSwitch
+"""
```

Comment:
@jirikotlin, it would be good to have some documentation here, and a `.. versionadded:: Boron` is needed as well.